### ISSUE TITLE
Add support for circles and arcs in symbols

### ISSUE THREAD

**Issue by Bouni** [uPesy/easyeda2kicad.py#45]
By now circles and arcs are not supported which leads to wierd symbles sometimes, here's nan example for `C318884`

Easyeda symbol:
![image](https://user-images.githubusercontent.com/948965/172818572-77b3feed-0401-44f1-8454-17eded446a64.png)

KiCAD symbol generated by this tool:
![image](https://user-images.githubusercontent.com/948965/172818267-e6fcef5d-b7b4-4d5b-a4f2-9cbf133d3d54.png)

According to the code, circles and arcs are not yet implemented:

- https://github.com/uPesy/easyeda2kicad.py/blob/6ac7bae3363c5a5cd185f7e55c24cc9e2b117bbe/easyeda2kicad/easyeda/parameters_easyeda.py#L143-L152

**Comment by Bouni** [comment]
Nice, circles seem to work in the latest dev commit 😍 

![grafik](https://user-images.githubusercontent.com/948965/173287704-62132a09-36ca-464c-b9e7-82dcde7cccdb.png)

![grafik](https://user-images.githubusercontent.com/948965/173287742-66a8b789-7369-4628-88af-f352fc9b1a69.png)

**Comment by uPesy** [comment]
Yes but i'm not sure if i should enable filling the background (having the circle with yellow background)
The circle are descrided as ellipse in the easyeda footprint 🤔

**Comment by Bouni** [comment]
Does the easyeda data show if their symbol has filled circles or not?

**Comment by uPesy** [comment]
No i don't think so

**Comment by Bouni** [comment]
In that case I would no fill them in ...

**Comment by uPesy** [comment]
Yeah you're right, but in easyeda, there is no background filling on symbol, contrary to Kicad :
![image](https://user-images.githubusercontent.com/61392244/173754686-9a5716b4-c658-4700-8603-fe9e75a9fb2d.png)
![image](https://user-images.githubusercontent.com/61392244/173754666-d3d2966a-247e-44fc-8604-6274be7b4415.png)

**Comment by Bouni** [comment]
For `C318884`: background fill in circles is gone, looks good to me!

![image](https://user-images.githubusercontent.com/948965/173792673-b471eb1e-9157-415d-a585-bd75c4bc13c0.png)

The optocoupler issue is answered in #53

**Comment by uPesy** [comment]
Symbol arc is now supported in the last version v0.5.0.
Note that easyeda use elleptical arc while Kicad support only circle arc.

**Comment by Bouni** [comment]
Are you aware of a symbol that use actual elliptical arcs?

I mean they most likely use elliptical arcs as "normal" arcs but never as an elliptical shape, right?

**Comment by uPesy** [comment]
For the coil, the radius is `rx=3.9` and `ry=4.0` on Easyeda

> I mean they most likely use elliptical arcs as "normal" arcs but never as an elliptical shape, right?
Yes

**Comment by Bouni** [comment]
What do you plan to do in such cases? 
I could imagine having a threshold for the difference in `rx` and `ry` of lets say 0.1mm and if that threshold is not exceeded use one of the values as arc radius. That way we end up wit usable symbols that might not fit a 100% but still better than skip the arcs completely.

**Comment by Bouni** [comment]
Maybe easier as my suggestion. Thinks that'll do the job.
